parlai/tasks/quac/build.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+# Download and build the data if it does not exist.
+
+import parlai.core.build_data as build_data
+import os
+import json
+
+VERSION = '0.2'
+TRAIN_FILENAME = 'train_v' + VERSION + '.json'
+VALID_FILENAME = 'val_v' + VERSION + '.json'
+
+URL = 'https://s3.amazonaws.com/my89public/quac/'
+SHOULD = '__SHOULD__'
+MAYBE = '__MAYBE__'
+SHOULD_NOT = '__SHOULDNOT__'
+
+YES = '__YES__'
+NO = '__NO__'
+NEITHER = '__NEITHER__'
+
+MAP_CONTINUATION = {'m': MAYBE, 'f': SHOULD, 'n': SHOULD_NOT}
+MAP_AFFIRMATION = {'y': YES, 'n': NO, 'x': NEITHER}
+
+OUTPUT_FORMAT = (
+    'text:{question}\tfollowup:{continuation}\tyesno:'
+    '{affirmation}\tanswer_starts:{start}\tlabels:{labels}'
+)
+
+
+def _parse_answers(q_a):
+    starts = []
+    labels = []
+    starts.append(str(q_a['orig_answer']['answer_start']))
+    labels.append(q_a['orig_answer']['text'].replace('|', ' __PIPE__ '))
+    for each in q_a['answers']:
```

Comment:
i would guess that 'orig_answer' is contained in 'answers' (from a quick glance at the data this appears to be true)--  can you confirm this @dexterju? in which case, we would not need to add 'orig_answer' to the labels.

(as a note, it's not necessary for 'orig_answer' to come first -- during training, most agents select a label randomly, and during evaluation, we check against all labels 😄)

Reply:
AHa, ok, thanks for telling me that! 
Yes, for many examples I saw, the `orig_answer` is included in `answers`, most of examples I saw have only one answer, so the two are the same.

Reply:
```
[{'text': 'The modern version of Kara Zor-El made her debut in Superman/Batman #8 (2004', 'answer_start': 1126}, {'text': 'Executive Editor Dan DiDio had been looking for a way to simplify the Supergirl character from her convoluted post-crisis history;',
 'answer_start': 59}, {'text': 'found an opening to reintroduce the character following the conclusion of the first story arc of Superman/Batman.', 'answer_start': 281}, {'text': 'The modern version of Kara Zor-El made her debut in Superman/Batman #8 (200
4).', 'answer_start': 1126}, {'text': 'Executive Editor Dan DiDio had been looking for a way to simplify the Supergirl character', 'answer_start': 59}]
{'text': 'Executive Editor Dan DiDio had been looking for a way to simplify the Supergirl character', 'answer_start': 59}
```
You are right, I will remove orig_answer